sanic/handlers.py

```diff
@@ -47,9 +61,13 @@ def response(self, request, exception):
         :param exception: Exception to handle
         :return: Response object
         """
-        handler = self.handlers.get(type(exception), self.default)
+        handler = self.lookup(exception)
         try:
-            response = handler(request=request, exception=exception)
+            response = handler and handler(
```

Comment:
About handler, handler will never be `_missing` at the point. In `lookup`, `handler` is replaced to other value as soon as it is checked as `_missing`. Please tell me again if you saw any `TypeError` there.

I can rewrite the `and` part like:
```None if handler is None else handler(...```
I thought it is a common shorthand, but if you feel so, changing would be better.